.travis.yml

```diff
@@ -56,9 +56,6 @@ matrix:
       env:
         - OS_TYPE=ubuntu_docker
         - OS_VERSION=bionic
-    - compiler: "gcc"
```

Comment:
similarly:
for this change it would be a good idea to add a small comment about why it's being removed (i.e. Xcode 6.4 has been deprecated, and will be dropped by Travis-CI in the future, etc, etc, etc)

Reply:
the same: added some details in the commit message and the PR description.